### ISSUE TITLE
The MultiSelect Clear button receives undefined "this" after transpiling with some Webpack versions.

### ISSUE THREAD

**Issue by simonssspirit** [telerik/kendo-react#64]
The MultiSelect clear button receives undefined this after pre-compiling with some web pack versions.

This is likely to occur due to the way the functional components are transpiling.

More details on tickets: 1338651, 1338666 and 1338069

**Comment by epavan** [comment]
Yep opened thorugh telerik support. Initially I thought it was a storybook issue cause I'm using storybook 4.0.0-alpha.16 which is still unstable but I get the same problem running the component in a normal app.

webpack: latest
"@progress/kendo-react-dropdowns": "latest",

**Comment by nstoychev** [comment]
@epavan - we have published a fix in our development channel, under a `dev` tag. Please refer to the following [article](https://www.telerik.com/kendo-react-ui/components/installation/development-builds/) on how to obtain the development builds.

**Comment by nstoychev** [comment]
We have planned it for the beginning of the next week. I will update this issue once we release.

**Comment by nstoychev** [comment]
@epavan - version 1.3.0 is live. Release notes can be found here: https://www.telerik.com/kendo-react-ui-develop/components/changelogs/ui-for-react/.
